ide/o.apache.xml.resolver/build.xml

```diff
@@ -49,8 +49,7 @@
             patchfile="external/xml-resolver-1.2-netbeans.patch" />
         <!-- ... 4. Compiles sources to build/external-patch/classes -->
         <mkdir dir="build/external-patch/classes" />
-        <javac srcdir="build/external-patch/sources"
-            destdir="build/external-patch/classes" source="${javac.source}" target="${javac.target}">
+        <javac srcdir="build/external-patch/sources" destdir="build/external-patch/classes" release="8">
```

Comment:
this looks weird but I hope that we can do a simple search and replace from
`release="8"` to
`release="${javac.release}"` in future once we change the format to feature version (instead of `1.x`).

I also would like to set the property by default to 8 and remove it from all module `project.properties` so that it can be managed centrally. Modules can still set it if they diverge from the default, but they don't have to set the default.